target/linux/ath79/dts/ar9344_atheros_db120.dts

```diff
@@ -0,0 +1,210 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9344.dtsi"
+
+/ {
+	model = "Atheros DB120 reference board";
+	compatible = "atheros,db120", "qca,ar9344";
+
+	aliases {
+		led-boot = &led_system;
+		led-failsafe = &led_system;
+		led-running = &led_system;
+		led-upgrade = &led_system;
+	};
+
+	leds: leds {
+		compatible = "gpio-leds";
+
+		wlan2g {
+			label = "db120:green:wlan2g";
```

Comment:
model prefix "db120:" has been removed due to uselessness. You may just change to green:wlan2g etc.

Reply:
Indeed, the patch was based on trunk at 5.4.71, I'll change this.